modules/installation-custom-ibm-cloud-vpc.adoc

```diff
@@ -32,13 +44,35 @@ include::snippets/custom-dns-server.adoc[]
 [id="installation-custom-ibm-cloud-vpc-validation_{context}"]
 == VPC validation
 
-The VPC and all of the subnets must be in an existing resource group. The cluster is deployed to this resource group.
+The VPC and all of the subnets must be in an existing resource group. The cluster is deployed to the existing VPC.
 
 As part of the installation, specify the following in the `install-config.yaml` file:
 
-* The name of the resource group
-* The name of VPC
-* The subnets for control plane machines and compute machines
+ifndef::ibm-cloud[]
+* The name of the existing resource group that contains the VPC and subnets
+endif::ibm-cloud[]
+ifdef::ibm-cloud[]
+* The name of the existing resource group that contains the VPC and subnets (`networkResourceGroupName`)
+endif::ibm-cloud[]
+ifndef::ibm-cloud[]
+* The name of the existing VPC
+endif::ibm-cloud[]
+ifdef::ibm-cloud[]
+* The name of the existing VPC (`vpcName`)
+endif::ibm-cloud[]
+ifndef::ibm-cloud[]
+* The subnets that were created for control plane machines and compute machines
+endif::ibm-cloud[]
+ifdef::ibm-cloud[]
+* The subnets that were created for control plane machines and compute machines (`controlPlaneSubnets` and `computeSubnets`)
+endif::ibm-cloud[]
```

Comment:
I recall tinkering with this and couldn't decide if it was better to have a pile of ifndefs/idefs or go with your suggestion. I landed on this because, IMO, the published output reads easier when the parameter is on the same line of the bullet. Once this merges, I plan on coordinating with Alisha/Silke to get the Power VS equivalent parameter names. At which point I see this section as:

ifdef:ibm-cloud[]
* Item (parameter)
* Item (parameter)
* ...
endif:ibm-cloud[]

ifdef:ibm-power-vs[]
* Item (parameter)
* Item (parameter)
* ...
endif:ibm-power-vs[]